app/src/components/Header.tsx

```diff
@@ -1,3 +1,5 @@
+"use client";
```

Comment:
We need to use client here b/ c the truss stuff?

Reply:
This component uses `useState` for hiding/showing the mobile menu